src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4075,6 +4074,7 @@ namespace VariableSpeedCoils {
                         SupTemp -= FanCoolLoad / (CpAir * rhoair * VolFlowRate);
                     }
                     MixWetBulb = PsyTwbFnTdbWPb(MixTemp, MixHumRat, OutBaroPress, RoutineName);
+                    // need to use OutTemp for air-cooled and RatedInletWaterTemp for water-cooled
                     TotCapTempModFac =
                         CurveValue(VarSpeedCoil(DXCoilNum).MSCCapFTemp(VarSpeedCoil(DXCoilNum).NormSpedLevel), MixWetBulb, RatedInletWaterTemp);
```

Comment:
I thought we had changed this line (for zone equipment and next line for air loop equipment) to account for air-cooled versus water-cooled. This needs to be corrected eventually. Need to compare diff's to #6830 before anything else happens.

Reply:
Is this still an open question?

> I thought we had changed this line (for zone equipment and next line for air loop equipment) to account for air-cooled versus water-cooled. This needs to be corrected eventually. Need to compare diff's to #6830 before anything else happens

Reply:
This needs correcting. I can do it here or elsewhere.

Reply:
That's your call - just let me know if you plan to add it here.